### PR TITLE
Update ChangeLog for 5.3.0.

### PR DESCRIPTION
**Pull request by interwq**

### CODE REVIEW

File: ChangeLog
Comment (Lapenkov) [inline ChangeLog:7]: Just in case - don't forget to change it.

**Comment by Lapenkov** [comment]
It doesn't say anything about the `san_guard` features. Is this because they are still "unofficial"?

File: ChangeLog
Comment (interwq) [reply ChangeLog:7]: I'll make sure to update before merging. Still waiting for Travis CI to get back, which may take a day or two.

**Comment by interwq** [comment]
> It doesn't say anything about the san_guard features. Is this because they are still "unofficial"?

Right. The sanitization, huge page allocator, and several other things will remain experimental and undocumented for now.

**Review by Lapenkov — APPROVED** [review]

**Pull request merged**
